lib/CannedPolicy.js

```diff
@@ -15,19 +14,56 @@ function CannedPolicy(url, expireTime) {
  *
  * @return {String} Serialized policy
  */
-CannedPolicy.prototype.toJSON = function() {
-  var policy = {
-    'Statement':[{
-      'Resource': this.url,
-      'Condition': {
-        'DateLessThan': {
-          'AWS:EpochTime': this.expireTime
-        }
-      }
-    }]
-  };
+CannedPolicy.prototype.toJSON = function () {
+    if (this._validate()) {
+        var policy = {
+            'Statement': [{
+                'Resource': this.url,
+                'Condition': {
+                    'DateLessThan': {
+                        'AWS:EpochTime': Math.round(
+                            this.expireTime.getTime() / 1000
+                        )
+                    }
+                }
+            }]
+        };
+    }
+    return JSON.stringify(policy);
+};
+
+/**
+ * Check for common mistakes with types
+ * @private
+ */
+CannedPolicy.prototype._validate = function () {
+    if (typeof this.expireTime === 'undefined') {
```

Comment:
Yes. Good catch. I fixed this throughout to standardize on 2-space indent (df26f4e7af130c8dd7de73f78d513d8a55fb1105)